### PR TITLE
Add schema

### PR DESCRIPTION
**Pull request by gideonthomas**
To test, just run `make schema`

@alanmoo r?

### CODE REVIEW

**Comment by alanmoo** [comment]
My gut says to `.gititnore` the schema image, as it's the equivalent of committing compiled code. Having instructions to run the command is helpful, but even with the disclaimer, I can see this getting out of sync pretty easily.

**Comment by alanmoo** [comment]
Other than that, this looks good

**Comment by gideonthomas** [comment]
@alanmoo ok, instead, I'll modify the disclaimer to say "commit the file" and I'll add a migration hook that will auto-generate it when you run `makemigrations` (which you have to for your PR to be valid when you make a change to the model).

File: app/manage.py [hunk 1]
```diff
@@ -7,4 +7,20 @@
 
     from django.core.management import execute_from_command_line
 
-    execute_from_command_line(sys.argv)
+    args = sys.argv
+
+    execute_from_command_line(args)
+
+    if args[1] == "makemigrations":
```
Comment (alanmoo) [inline app/manage.py]: For the sake of understanding, is this intercepting and injecting some commands when this argument is passed, then continuing on elsewhere?

Comment (gideonthomas) [inline app/manage.py]: Actually it's executing the command first (line 12) and then checking what command was executed and then executing another command after that

Comment (alanmoo) [inline app/manage.py]: Ah now I see that. Thanks!

**Pull request closed without merging**